composer.json

```diff
@@ -91,7 +91,9 @@
 
         "matthimatiker/opcache-bundle": "^0.7",
         "qandidate/toggle-bundle": "^1.1.0",
-        "google/recaptcha": "~1.1"
+        "google/recaptcha": "~1.1",
+        "ggggino/wordbundle": "^1.0",
```

Comment:
I think this bundle is not really needed, it doesn't add much and makes things a bit more complicated.